app/src/main/java/com/example/partyPlayer/MainActivity.kt

```diff
@@ -109,7 +106,8 @@ class MainActivity : AppCompatActivity() {
                 try {
                     println("--------------------------------------------------------------------$response")
 
-                } catch (e: JSONException) {
+                }
+                catch (e: JSONException) {
```

Comment:
Moved the catch statement off of the same line as the try statement ending curly brace

Reply:
The location of curlies has been a long debated topic in programming circles for decades.  Either way, it's best to be consistent: if you have a separate line for curlies, put them all on a separate line.  in other words, move the end curly on line 110 to a new line.

All in all, putting curlies on the same line as the catch appears to be convention.